README.md

```diff
@@ -54,11 +54,7 @@ your own repository.
    Note: The application name is wrapped around comments used to disable a rubocop check.
    When the application has been renamed, the comments should be removed.
    (See config/application.rb#L25-27)
-4. `cp .env.example .env`
-5. `cp .env.development.example .env.development`
-6. `cp .env.test.example .env.test`
-7. `mv README.example.md README.md`
-8. Add `export DATABASE_URL="postgresql://HOST:@localhost:5432"` to .env.development and .env.test
+4. `mv README.example.md README.md`
```

Comment:
```suggestion
1. `mv README.example.md README.md`
```

>__<

Reply:
I think there are pros and cons of making this change:

Pros:
It will be easier to add more stuff later.

Cons:
The raw file will be less readable.

Personally I prefer the numbers to be correct.

Reply:
ahhh, I didn't see that it was one list, never mind me 💃